### PR TITLE
Fixed footer issues error 

### LINKED ISSUE #124: [BUG] Change copyright message
### 🙁 Actual behavior

© 2023 BhagavadGita.io

### 🙂 Expected behavior

Should be:
© 2023 Copyright: [Ved Vyas Foundation](https://vedvyas.org). All rights reserved.

Year should be dynamic and not hardcoded.

### 🔢 Steps to Reproduce the Problem

_No response_

### PR DESCRIPTION
**Pull request by eht-ck**
fixed: Syntax error in the footer and open in new tab functionality to the about-us,blog,donate,press,api and contact-us
Fixed : #124 #127 #129 
Description: 
Getting  syntax error when setting up repo
`
Caused by:
    0: failed to process input file
    1: Syntax Error
wait  - compiling /_error (client and server)...
error - ./components/Footers/Footer.js
Error:
  x Expected ',', got '{'
    ,----
 12 | {
    : ^
    `----
`
Fixed the syntax and added opening the external links in footer links to new tab and solved 3 issues #124 #127 #129 
image

### Summary by OpenAI

```
**Refactor:**
- Updated URLs for blog, donate, api, and contact-us links in the footer.
- Removed Instagram link from the footer.
- Updated icons for Google Play and App Store links.
- Added `rel` attributes to links for security improvements.
- Updated footer text and layout for better user experience.

**Bug Fix:**
- Resolved issues #124, #127, and #129 related to footer functionality.
```

> 🎉 Here's to the changes we've made,  
> To the footer that's been skillfully upgraded.  
> No more bugs, no more fuss,  
> Just a smooth ride on our website bus. 🚌  
> So here's to the team, hip hip hooray!  
> We've made things better in every way. 🥳

### CODE REVIEW

**Comment by samanyougarg** [comment]
Thank you for the PR @eht-ck!

**Comment by samanyougarg** [comment]
- [x] These buttons should be properly aligned and of the same height and width.

Screenshot 2023-07-02 at 12 03 57 AM

**Comment by samanyougarg** [comment]
- [x] Make the ved Vyas link in orange color same as our theme. Also, this link should open in a new tab.

Screenshot 2023-07-02 at 12 04 45 AM

**Comment by samanyougarg** [comment]
- [ ] Remove the Instagram link as we don't have an official account.

File: components/Footers/Footer.js [hunk 1]
```diff
@@ -5,15 +5,24 @@ import classNames from "../../utils/classNames";
 /* This example requires Tailwind CSS v2.0+ */
 const navigation = {
   main: [
-    { name: "About Us", href: " /about " },
-    { name: "Blog", href: "https://radhakrishna.net/" },
-    { name: "Donate", href: "https://opencollective.com/the-gita-initiative" },
-   {/*  { name: "Press", href: "" }, */}
+    { name: "About Us", href: " /about ", target: "_blank" },
```
Comment (samanyougarg) [inline components/Footers/Footer.js]: This should open in the same tab since it is an internal link.

File: components/Footers/Footer.js [hunk 2]
```diff
@@ -5,15 +5,24 @@ import classNames from "../../utils/classNames";
...
+    { name: "Blog", href: "https://radhakrishna.net/", target: "_blank" },
+    {
+      name: "Donate",
+      href: "https://opencollective.com/the-gita-initiative",
+      target: "_blank",
+    },
+    { name: "Press", href: "", target: "_blank" },
```
Comment (samanyougarg) [inline components/Footers/Footer.js]: Hide this for now since we dont have a Press link.

File: components/Footers/Footer.js [hunk 3]
```diff
@@ -5,15 +5,24 @@ import classNames from "../../utils/classNames";
...
     {
       name: "API",
       href: "https://rapidapi.com/bhagavad-gita-bhagavad-gita-default/api/bhagavad-gita3",
+      target: "_blank",
+    },
+    {
+      name: "Contact Us",
```
Comment (samanyougarg) [inline components/Footers/Footer.js:19]: This is an email link should open in the same tab.

**Review by samanyougarg — CHANGES_REQUESTED** [review]

**Comment by eht-ck** [comment]
image

image

Actually  aspect ratio of play store and ios images are different, so if width of both are made same height will be different and vice versa. 

So cropping the sides of play store logo is the only solution which I found to make height and width same.

Please suggest any other solution if you have to do it without cropping the playstore logo

**Comment by samanyougarg** [comment]
- [x] Too much gap between the buttons.

Screenshot 2023-07-08 at 12 36 14 AM

**Comment by samanyougarg** [comment]
- [x] Let's use SVGs. These are of the same size.

app-store
google-play

**Comment by samanyougarg** [comment]
- [ ] Also, please see the AI reviewer comments above. @eht-ck

**Comment by eht-ck** [comment]
image
image
Fixed the size and spacing between app  imgs

**Comment by eht-ck** [comment]
image
image
Fixed the size and spacing between app  images and tried to resolve relevant suggestion of AI reviewer

**Comment by samanyougarg** [comment]
@eht-ck Distance between the buttons keeps increasing on higher resolutions. Try increasing the display zoom to 125% and then test.

Screenshot 2023-07-08 at 12 35 59 PM

**Comment by eht-ck** [comment]
> @eht-ck Distance between the buttons keeps increasing on higher resolutions. Try increasing the display zoom to 125% and then test.
> 
> Screenshot 2023-07-08 at 12 35 59 PM
It is correct on my system
image
image
image

**Comment by samanyougarg** [comment]
Sorry, I meant if you zoom out. On bigger screens, it is like this.

**Comment by samanyougarg** [comment]
Also, please clean up the PR to remove the other image files that are not needed anymore.

**Comment by eht-ck** [comment]
@samanyougarg Fixed the image gap issue  for high resolution screen and removed other images  files from public directory which arent being used.
image

**Comment by samanyougarg** [comment]
Looks good, thank you!

**Review by samanyougarg — APPROVED** [review]

**Pull request merged**